src/core/server/logging/README.mdx

```diff
@@ -659,6 +659,23 @@ and you can enable them by adjusting the minimum required [logging level](#log-l
 #### logging.filter
 TBD
 
+#### logging.rotate
+Specify the options for the logging rotate feature and only applicable when logs are written to file.
+With the new logging config, the log rotation feature is provided by the `rolling-file` [appender](#rolling-file-appender).
+
+**`logging.rotate.enabled` and `logging.rotate.usePolling`**
+Enables log rotation when `enabled` is set to `true`. The `usePolling` option is optional and can be used in systems where the watch api is not accurate.
+With the new logging config log rotation is provided by the rolling file appender. Polling will apply by default when the `rolling-file` appender is configured.
+
+**`logging.rotate.pollingInterval`**
+The number of milliseconds for the polling strategy in the case when `logging.rotate.usePolling` is enabled and defaults to 10000.
+Possible range from 5000 to 3600000. With the new logging config an time interval can be configured with the
+[TimeIntervalTriggeringPolicy](#timeintervaltriggeringpolicy)
```

Comment:
The `rolling-file` appender documentation is great and I didn't want to be too repetitive in these sections.